src/components/dashboard/send-money.svelte

```diff
@@ -42,6 +42,7 @@
       type="number"
       isWithItem
       isItemRight
+      maxlength={8}
```

Comment:
it has `type="number"`. Shouldn't this be something like `max={99999999}` instead?

Reply:
Doesn't really matter